Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageTextRenderer.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+/// The component responsible for getting the text content of a message text.
+struct ChatMessageTextRenderer {
+    func text(for message: ChatMessage, channel: ChatChannel) -> String {
```

Comment:
Do we need the `channel` here? Another alternative could be to provide optional language (and default to the system language if not provided). That way we could use it in `textContent`, without deprecating. Wdyt?

Reply:
Yeah that might be a good idea 👍 I'll look into this

Reply:
Shall we rename this one to `ChatMessageTextFormatter`?